docker/webapp_config.json

```diff
@@ -1,6 +1,6 @@
 {
   "database_data": {
-    "local":{
+    "local": {
```

Comment:
Sorry about the unrelated format changes here - but since the whole file was using inconsistent spacing and style, I felt it'd clearer for everyone to normalise it.

I can revert them if you'd like, no problem.